.github/services/s3/ceph_rados_s3/action.yml

```diff
@@ -0,0 +1,79 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: ceph_rados_s3
+description: 'Behavior test for CEPH OBJECT GATEWAY S3.'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup Ceph Rados Server
+      shell: bash
+      working-directory: fixtures/s3/ceph-rados
+      run: |
+        # Create Docker network
+        echo "Creating Docker network..."
+        docker network create --driver bridge ceph-cluster-net
+        
+        # Start Ceph monitors and manager
+        docker-compose up -d mon1 mgr
+        
+        # Update Ceph configuration
+        echo "Updating Ceph configuration..."
+        echo "osd pool default min size = 2" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "max open files = 655350" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "cephx cluster require signatures = false" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "cephx service require signatures = false" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "osd max object name len = 256" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        echo "osd max object namespace len = 64" | sudo tee -a ~/volumes_data/ceph/ceph/ceph.conf
+        
+        # Restart monitors and manager
+        echo "Restarting monitors and manager..."
+        docker-compose restart mon1 mgr
+        
+        # Start OSDs, RGW, and MDS
+        echo "Starting OSDs, RGW, and MDS..."
+        docker-compose up -d osd1 osd2 osd3
+        docker-compose up -d rgw1
+        docker-compose up -d mds1
+        
+        # Enable and configure Ceph Dashboard
+        echo "Enabling and configuring Ceph Dashboard..."
+        docker-compose exec -T mon1 ceph mgr module enable dashboard --force
```

Comment:
Do we really need dashboard? I'm guessing we can't access it.

Reply:
ok, got it, will delete